salt/beacons/__init__.py

```diff
@@ -170,19 +170,19 @@ def _process_interval(self, mod, interval):
         Process beacons with intervals
         Return True if a beacon should be run on this loop
         """
-        log.trace("Processing interval %s for beacon mod %s", interval, mod)
+        log.debug("Processing interval %s for beacon mod %s", interval, mod)
         loop_interval = self.opts["loop_interval"]
         if mod in self.interval_map:
-            log.trace("Processing interval in map")
+            log.debug("Processing interval in map")
             counter = self.interval_map[mod]
-            log.trace("Interval counter: %s", counter)
+            log.debug("Interval counter: %s", counter)
             if counter * loop_interval >= interval:
                 self.interval_map[mod] = 1
                 return True
             else:
                 self.interval_map[mod] += 1
         else:
-            log.trace("Interval process inserting mod: %s", mod)
+            log.debug("Interval process inserting mod: %s", mod)
```

Comment:
We should probably change all of these back to `trace`

Reply:
Done